src/components/map/NavigationButtons.vue

```diff
@@ -4,11 +4,13 @@ import { Viewpoint } from '@vcmap/core'
 import { inject, reactive } from 'vue'
 import IconHome from '../ui/icons/IconHome.vue'
 import UiButton from '../ui/UiButton.vue'
+import CompassComponent from './CompassComponent.vue'
+import NavigationHelp from './NavigationHelp.vue'
 
 const vcsApp = inject('vcsApp') as VcsApp
 
 const state = reactive({
-  is3D: vcsApp.maps.activeMap.name === 'cesium',
+  is3D: vcsApp?.maps?.activeMap?.name === 'cesium',
```

Comment:
CMIIW, will this become an issue because we use the attribute from the vcsApp as reactive?

If so, probably use the same approach as in "syncCompas" (i.e. listen to the state of the activeMap)

Reply:
IMHO I don't think its become an issue because it's rather the components that subscribe to the state that will be reactive not the vcsApp which is only the initial value.